helsinki/login/service-connection-confirm-form.ftl

```diff
@@ -1,21 +1,20 @@
 <#import "template.ftl" as layout>
 <@layout.registrationLayout; section>
     <#if section = "header">
-        FIXME: header
+      ${msg("weUseInformationTitle")}
     <#elseif section = "form">
-        <div>Service: ${serviceName}</div>
-        <div>
-            <ul>
-            <#list serviceAllowedData as item>
-                <li>${item}</li>
-            </#list>
-            </ul>
-        </div>
-        <form id="kc-service-connection-confirm-form" action="${url.loginAction}" method="post">
-            <div>
-                <button id="kc-accept" type="submit" name="response" value="accept">${msg("profileAcceptButtonText")}</button>
-                <button id="kc-decline" type="submit" name="response" value="decline">${msg("profileDeclineButtonText")}</button>
-            </div>
-        </form>
+      <p>${kcSanitize(msg("serviceDataUsageListTitleOldUser",serviceName))}</p>
+      <ul class="checked-list content-before-buttons">
+        <#list serviceAllowedData as item>
+          <li>${item}</li>
+        </#list>
+      </ul>
+      <form id="kc-service-connection-confirm-form" action="${url.loginAction}" method="post">
+          <div class="wide-buttons">
+              <button class="${properties.kcButtonClass!} ${properties.kcButtonPrimaryClass!} ${properties.kcButtonLargeClass!}" id="kc-accept" type="submit" name="response" value="accept">${msg("profileAcceptButtonText")}</button>
+              <button class="${properties.kcButtonClass!} ${properties.kcButtonDefaultClass!} ${properties.kcButtonLargeClass!}" id="kc-decline" type="submit" name="response" value="decline">${msg("profileDeclineButtonText")}</button>
```

Comment:
Some design has texts "Anna lupa" and "Älä anna lupaa" in the buttons. Is that part of the design obsolete or what should the button texts be?

Reply:
Design is an overwrite on the Tunnistamo version of the permission dialog. "Jatka" and "Peruuta" are used in about all views in Keycloak and approved texts in the buttons.

Reply:
I take that as to mean that we are not obeying the design to the point, but use the texts that the template currently has :smirk: